src/components/FunctionsPage/Functions.js

```diff
@@ -108,6 +108,17 @@ const Functions = ({
   const location = useLocation()
   const dispatch = useDispatch()
 
+  const functionsFilter = useMemo(
```

Comment:
```suggestion
  const functionsFilters = useMemo(
```

also please check and pass this const in every place where filters should be passed so we won't get such a bug again

Reply:
@mariana-furyk done.